libs/components/src/tabs/tab-item.component.html

```diff
@@ -0,0 +1,27 @@
+<ng-template #linkRef>
+  <a
+    role="tab"
+    [class]="baseClassList"
+    [routerLink]="route"
+    [routerLinkActive]="activeClassList"
+    #rla="routerLinkActive"
+    [attr.aria-selected]="rla.isActive"
+  >
+    <ng-container [ngTemplateOutlet]="contentRef"></ng-container>
+  </a>
+</ng-template>
+<ng-template #btnRef>
+  <button
+    type="button"
+    role="tab"
+    [class]="baseClassList"
+    [disabled]="disabled"
+    [attr.aria-disabled]="disabled"
+  >
+    <ng-container [ngTemplateOutlet]="contentRef"></ng-container>
+  </button>
+</ng-template>
+<ng-template #contentRef>
+  <ng-content></ng-content>
+</ng-template>
+<ng-container [ngTemplateOutlet]="disabled ? btnRef : linkRef"></ng-container>
```

Comment:
What do you think about using a switch here instead? Unfortunately I couldn't get rid of the last container. I also changed the disabled and aria-disabled to not be bindings to make it more obvious what's going on.

```suggestion
<ng-container [ngSwitch]="disabled">
  <a
    *ngSwitchCase="false"
    role="tab"
    [class]="baseClassList"
    [routerLink]="route"
    [routerLinkActive]="activeClassList"
    #rla="routerLinkActive"
    [attr.aria-selected]="rla.isActive"
  >
    <ng-content [ngTemplateOutlet]="content"></ng-content>
  </a>
  <button
    *ngSwitchCase="true"
    type="button"
    role="tab"
    [class]="baseClassList"
    disabled
    aria-disabled="true"
  >
    <ng-content [ngTemplateOutlet]="content"></ng-content>
  </button>
</ng-container>

<ng-template #content>
  <ng-content></ng-content>
</ng-template>

```

Reply:
Fixed a typo in your suggestion (using `ng-content` inside of the switch items instead of `ng-container` and this is working as expected and is cleaner. Thanks for the suggestion!